packages/@aws-cdk/aws-rds/lib/private/instance-deletion-policy.ts

```diff
@@ -0,0 +1,18 @@
+import { CfnDeletionPolicy, RemovalPolicy } from '@aws-cdk/core';
+import { CfnDBInstance } from '../';
+
+export function applyInstanceDeletionPolicy(cfnDbInstance: CfnDBInstance, removalPolicy: RemovalPolicy | undefined): void {
+  const instanceDeletionPolicy = toCfnDeletionPolicy(removalPolicy);
+  const instanceCfnOptions = cfnDbInstance.cfnOptions;
+  instanceCfnOptions.deletionPolicy = instanceDeletionPolicy;
+  instanceCfnOptions.updateReplacePolicy = instanceDeletionPolicy;
+}
+
+function toCfnDeletionPolicy(removalPolicy: RemovalPolicy | undefined): CfnDeletionPolicy {
```

Comment:
Weird that you split this up into its own method. I would have just done it inline with `applyInstanceDeletionPolicy()` but feel free to keep it if this is your style.

Reply:
This has been all changed with the introduction of `RemovalPolicy.SNAPSHOT`.